compiler/rustc_llvm/llvm-wrapper/CoverageMappingWrapper.cpp

```diff
@@ -1,9 +1,11 @@
 #include "LLVMWrapper.h"
+#include "llvm/ADT/ArrayRef.h"
```

Comment:
I've filed #123437 as a separate PR to format this file with `clang-format`.

After that is merged, you should be able to rebase this PR on top of it to get rid of the unrelated formatting changes.